swap_meet/item.py

```diff
@@ -1,2 +1,22 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition = 0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        default = "Hello World!"
+        return default
+    
+    def condition_description(self):
+        if self.condition == 5:
+            return(f"Absolutely Fabulous")
+        elif self.condition == 4:
+            return(f"Good enough")
+        elif self.condition == 3:
+            return(f"I'll take it")
+        elif self.condition == 2:
+            return(f"Decent")
+        elif self.condition == 1:
+            return(f"Meh")
+        elif self.condition == 0:
+            return(f"Burn it!")      
```

Comment:
This works well if the condition is an integer from 0-5. But what would happen if it were outside that range? Or if it were `3.5` (like it is in some of the tests)? How could you change your code to properly handle these cases?

Also, you don't need the put the `f` before any of these strings. We only use f-strings if there a variable or expression we want to put inside the string.